File: app/src/main/java/com/sofar/business/github/model/RepoSearchResponse.java
package com.sofar.business.github.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RepoSearchResponse {

  @SerializedName("total_count")
  int total;

  @SerializedName("items")
  List<Repo> items;

  int nextPage;
}

File: app/src/main/java/com/sofar/network/cache/model/RepoSearchResponse.kt
package com.sofar.network.cache.model

import com.google.gson.annotations.SerializedName

data class RepoSearchResponse(
  @SerializedName("total_count")
  val total: Int,

  @SerializedName("items")
  val items: List<Repo>,

  val nextPage: Int
)
